client/tests/acceptance/pas-form-validation-messages-display-test.js

```diff
@@ -155,4 +155,79 @@ module('Acceptance | pas form validation messages display', function(hooks) {
     await fillIn('[data-test-textarea="dcpProjectattachmentsotherinformation"]', 'abc');
     assert.dom('[data-test-validation-message="dcpProjectattachmentsotherinformation"]').doesNotExist();
   });
+
+  test('Certain Applicant fields on the PAS Form display both Saveable and Submittable validation errors', async function(assert) {
+    const project = this.server.create('project', 1, 'applicant');
+    const { pasForm } = project.packages.models[0];
+
+    this.server.create('applicant', 'individualApplicant', { pasForm });
+
+    this.applicants = [
+      await this.owner.lookup('service:store').findRecord('applicant', 1),
+    ];
+
+    await visit('/pas-form/1/edit');
+
+    // email
+    assert.dom('[data-test-validation-message="dcpEmail"]').doesNotExist();
```

Comment:
Doesn't email also have a "field is required" validation coming from `submittable-applicant-form.js`? Were you thinking of testing for that as well?

Reply:
@trbmcginnis  Great comment! Thank you. This made me also realize I should revise how I created the applicants within the tests. I've now added tests for the required validation, which should definitely be included according to the test title.